src/main/scala/tech/cryptonomic/conseil/util/EffectsUtil.scala

```diff
@@ -0,0 +1,26 @@
+package tech.cryptonomic.conseil.util
+
+import cats._
+import cats.arrow.FunctionK
+import cats.effect.IO
+import scala.concurrent.Future
+import slick.dbio.DBIO
+import scala.concurrent.ExecutionContext
+import cats.effect.ContextShift
+
+object EffectsUtil {
+
+  def toIO[T](future: => Future[T]): IO[T] = IO.fromFuture(IO(future))
+
+  /** Runs the db action converting the result to a lazy IO value */
+  def runDbIO[T](db: DatabaseUtil.DatabaseType, dbAction: DBIO[T])(implicit defaultContext: ContextShift[IO]): IO[T] =
```

Comment:
Could you explain why we convert DB executions from DBIO to Cats IO? Is there a performance benefit or degradation?

Reply:
For compatibility (with  other effectful operations) and lazyness (with respect to running DBIO as a Future, which is started eagerly, therefore is not totally under control)